frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/NotificationHandlerTest.java

```diff
@@ -96,6 +100,9 @@ void shouldSuccessfullyProcessResetPasswordConfirmationFromSQSQueue()
     @Test
     void shouldSuccessfullyProcessResetPasswordEmailFromSQSQueue()
             throws JsonProcessingException, NotificationClientException {
+        when(configService.getFrontendBaseUrl()).thenReturn(FRONTEND_BASE_URL);
+        when(configService.getCustomerSupportLinkRoute()).thenReturn(CUSTOMER_SUPPORT_LINK_ROUTE);
```

Comment:
Think you could avoid duplication by sticking these 2 lines in the setup method. 

`        when(configService.getFrontendBaseUrl()).thenReturn(FRONTEND_BASE_URL);
        when(configService.getCustomerSupportLinkRoute()).thenReturn(CUSTOMER_SUPPORT_LINK_ROUTE);`